.travis.yml

```diff
@@ -13,10 +13,9 @@ python:
   - "2.7"
 
 install:
-  pip install --user -r ./python/requirements.txt
-
+    -   pip install --user tox
 script:
-  - nosetests ./python -v --exclude-dir=./python/mleap/pyspark
+  - tox -c python/tox.ini -e $(echo py$TRAVIS_PYTHON_VERSION | tr -d .) -v

```

Comment:
isn't `tox -c python/tox.ini` sufficient? All version in envlist from the tox.ini will be run.
eg:
```
envlist = py27,py36
```

Reply:
the .travis is not working well with tox system. The workaround is to make travis install 2 different version of python and make tox run for each specific python version. 

I am using a solution mentioned here: https://stackoverflow.com/a/42309035

Note that there's an plugin https://github.com/tox-dev/tox-travis which will resolve the issue. However, the package assumes your tox.ini is on the root folder, and I am not sure if we want to move tox.ini out of python folder, because I make working folder to be `python`.   Also the old `run_test` script also seems to assume python is the working folder when running the tests, therefore I am not using this approach and just update the travis command to run tox instead.

Reply:
ok, makes sense